python/ray/dashboard/client/src/pages/dashboard/node-info/NodeInfo.tsx

```diff
@@ -92,31 +111,39 @@ class NodeInfo extends React.Component<
       };
     } = {};
 
+    const clusterWorkerPidsByIp = clusterWorkerPids(rayletInfo);
```

Comment:
High level comment describing what the following block is doing would be very helpful here